### PR TITLE
skypeforlinux: 5.4.0.1 -> 8.8.76.61628

### PR DESCRIPTION
**Pull request by bkchr**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 let
 
-  version = "5.4.0.1";
+  version = "8.8.76.61628";
```
Comment (Mic92) [inline pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: To my knowledge this is the testing version of skype.

Comment (bkchr) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: Hmm, arch is using the [same](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=skypeforlinux-bin). But you are right, it is still beta. Does this mean you want me to rename the package as "skypeforlinux-unstable"?

Comment (bkchr) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: The problem for me was that the old version could not be installed anymore.

Comment (Mic92) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: I just started skype and can login without problems. We could add a seperate version for unstable though.

Comment (bkchr) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: Which version? 5.4.0.1? Did you reinstalled it?

Comment (Mic92) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: 5.4.0.1 on nixpkgs master. I also removed all settings related to skype, but it still works.

Comment (bkchr) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: Ohh, I see what I have done wrong... I tried with nix-env and that does not use my nixpkg checkout :/ That tried to install 5.3.0.1 and that did not worked.

Comment (Mic92) [reply pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: `5.4.0.1` should be also in 17.09.

**Pull request closed without merging**